packages/dds/tree/src/test/forest/cursor.bench.ts

```diff
@@ -0,0 +1,100 @@
+/*!
```

Comment:
we want this file to be included in correctness tests (so correctness mode for benchmarking will include one iteration of the benchmark to make sure its working/passing. Thus I think naming it cursor.spec.bench.ts or modifying the mocha config to include .bench would be good.

If you do this I think you can also remove the path filter from the mocha invocation in the scripts, and just let it get that from its config file.

Reply:
Thanks.  It looks like the 'test:mocha' script specifies the path as `--recursive dist/test`.  I'll use the same for consistency.

(I did verify that the benchmark runs a single iteration w/'npm t')